src/planner.h

```diff
@@ -40,7 +40,8 @@ ts_create_private_reloptinfo(RelOptInfo *rel)
 static inline TimescaleDBPrivate *
 ts_get_private_reloptinfo(const RelOptInfo *rel)
 {
-	return rel->fdw_private;
+	Assert(rel->fdw_private);
```

Comment:
If the assert would be always true then the if/else isn't needed. The problem was that this condition is apparently *not* always true, in which case it'll still abort in debug builds?

Reply:
if we end up in the assert here that means this hypertable was not expanded by timescaledb code but by normal postgres table inheritance which is not the end of the world but on oversight and something that is useful to detect in debug build because that is probably on oversight on our part.